src/main/java/pl/betoncraft/betonquest/StaticEvents.java

```diff
@@ -58,12 +58,16 @@ public StaticEvents() {
                     LogUtils.getLogger().log(Level.WARNING, "Incorrect time value in static event declaration (" + key + "), skipping this one");
                     continue;
                 }
-                LogUtils.getLogger().log(Level.FINE, "Scheduling static event " + value + " at hour " + key + ". Current timestamp: "
+                LogUtils.getLogger().log(Level.FINE, "Scheduling static events " + value + " at hour " + key + ". Current timestamp: "
                         + new Date().getTime() + ", target timestamp: " + timeStamp);
-                // add the timer to static list, so it can be canceled if needed
                 try {
-                    TIMERS.add(new EventTimer(timeStamp, new EventID(pack, value)));
-                } catch (ObjectNotFoundException e) {
+                    final String[] events = value.split("[,]");
```

Comment:
Useless character class for one character:
```suggestion
                    final String[] events = value.split(",");
```

Reply:
fixed